README.md

```diff
@@ -67,6 +65,17 @@ class KarhooConfig(val context: Context): KarhooUISDKConfiguration {
     override fun authenticationMethod(): AuthenticationMethod {
         return AuthenticationMethod.KarhooUser()
     }
+
+    override suspend fun requireSDKAuthentication(callback: () -> Unit) {
+       KarhooApi.userService.logout()
```

Comment:
This example is not up to date and does not reflect the external docs. Please review it.